test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -220,8 +220,13 @@ thread_func_checkpoint(void *arg)
 {
     struct thread_data *td;
     WT_SESSION *session;
+    time_t t;
+    uint64_t sleep_sec;
+    int i;
     bool signalled;
 
+    srand((u_int)time(&t));
```

Comment:
`srand` is a bad RNG, but I'd agree we probably don't care much here, using srand/rand is fine.

Reply:
All other tests, including many in `test/csuite` use `__wt_random`. This should too for consistency I think.

Reply:
Changed to `__wt_random_init_seed/__wt_random`. Thank you!